### PR TITLE
AWS SNS: updating to SDK 2.x

### LINKED ISSUE #1343: AWS SNS: Use AWS SDK 2.0
See #372

### PR DESCRIPTION
**Pull request by gabfssilva**
Fixes #1343 

See also #372 and #1450

### CODE REVIEW

File: sns/src/main/scala/akka/stream/alpakka/sns/SnsPublishSettings.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.sns
+
+class SnsPublishSettings(val concurrency: Int) {
```
Comment (2m) [inline sns/src/main/scala/akka/stream/alpakka/sns/SnsPublishSettings.scala]: Make the constructor private and the class final.

Comment (gabfssilva) [reply sns/src/main/scala/akka/stream/alpakka/sns/SnsPublishSettings.scala]: fixed

**Comment by 2m** [comment]
There is a merge commit in this PR's branch. Can you remove it? Conflicts with latest master should be solved by rebasing the PR branch, if there are any.

**Comment by gabfssilva** [comment]
sorry, fixed. ;)

**Pull request merged**